test/scripts/checks/licenses.sh

```diff
@@ -2,4 +2,5 @@
 
 source src/dev/ci_setup/setup_env.sh
 
-yarn run grunt run:licenses
```

Comment:
Would you mind removing the `run:licenses` config from grunt while you're at it?

Reply:
It's still used by `jenkins:unit` called by `test/scripts/jenkins_unit.sh` at the moment.

Reply:
That script is still used for es verification, but yeah... I guess we can just wait for brian to get the rest of the jobs migrated to tasks so we can clean up all this old grunt nonsense